src/TimeData.cs

```diff
@@ -1,20 +1,15 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-
-namespace FluentBehaviourTree
+﻿namespace FluentBehaviourTree
```

Comment:
Whoops. This and the src/BehaviourTreeStatus.cs changes were just my IDE removing unused using statements. I'll add those back in.